compass/ocean/tests/overflow/hydro_vs_nonhydro/namelist.forward

```diff
@@ -0,0 +1,19 @@
+config_write_output_on_startup = .true.
+config_run_duration = '0000_03:00:00'
+config_dt = '00:00:0.5'
+config_time_integrator = 'RK4'
+config_use_mom_del2 = .true.
+config_mom_del2 = 1.0
+config_use_mom_del4 = .false.
+config_vert_tracer_adv_flux_order = 3
+config_horiz_tracer_adv_order = 2
+config_coef_3rd_order = 0.25
+config_flux_limiter = 'monotonic'
+config_remap_limiter = 'monotonic'
+config_enable_nonhydrostatic_mode = .false.
+config_use_cvmix = .true.
+config_disable_vel_coriolis = .false.
```

Comment:
```suggestion
```
I think the default is `.false.` so you shouldn't need this line

Reply:
Done.